cmd/wit-bindgen-go/main_test.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"testing"
+)
+
+// TestSimpleGenVerbosity ensures that a basic generation case honors the verbose flag
+func TestSimpleGenVerbosity(t *testing.T) {
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+	cmd := Command
+	cmd.Writer = &stdout
+	cmd.ErrWriter = &stderr
+
+	args := []string{
+		"wit-bindgen-go",
+		"generate",
+		"--world",
+		"http-fetch-simple",
+		"--dry-run",
+		"../../testdata/codegen/simple-http.wit",
+	}
+
+	// Run the app without verbose
+	cmd.Run(context.Background(), args)
+	if stderr.Len() != 0 {
+		t.Errorf("output was written to stderr despite lack of --verbose")
+	}
+
+	stdout.Reset()
+	stderr.Reset()
+
+	// Run the app WITH verbose
+	cmd.Run(context.Background(), append(args, "-v"))
+	if stderr.Len() == 0 {
+		t.Errorf("no output was written to stderr when --verbose was used")
+	}
```

Comment:
**suggest**: now the test can be expanded to test -v and -vv.

Reply:
The error message should probably reflect the flag used, too.

Reply:
Thanks for the suggestions, updated to test all 3 variations (`-v`, `-vv` and `--verbose`)!